app/create.js

```diff
@@ -43,18 +43,19 @@ define(['app/config', 'app/database', 'jquery', 'bootstrap', 'app/select', 'app/
 		})
 	}
 
-  // Auxiliary replace functions
-  function giveId(string, formItemId) {
-    return string.replace(/{i}/g, formItemId);
-  }
-  function giveRequired(string) {
-    return string.replace(/{required}/g, 'required=""');
-  }
-
 	// Auxiliary form functions
   function getItemVal(formName, formIndex) {
     return $("#items input[name='" + formName + formIndex + "']").val();
   }
+	function getFormVal(parentName, formType, formName) {
+    return $(parentName).find(formType + '[name="' + formName + '"]').val();
+  }
+
+	// Function for showing the user the system is loading
+	function showLoading(onSuccess) {
+		$("#loadFrame").children("h1").html("Creating dataset...")
+		$("#loadFrame").fadeIn(300, onSuccess);
```

Comment:
Shouldn't this 300 be some constant? If it is only used in here, you might want to name it fadeInTime or something like that.